src/site/stages/build/process-cms-exports/schemas/transformed/paragraph-spanish_translation_summary.js

```diff
@@ -0,0 +1,20 @@
+module.exports = {
+  type: 'object',
+  properties: {
+    contentModelType: { enum: ['paragraph-spanish_translation_summary'] },
+    entity: {
+      type: 'object',
+      properties: {
+        entityType: { enum: ['paragraph'] },
+        entityBundle: { enum: ['spanish_translation_summary'] },
+        fieldTextExpander: { type: 'string' },
+        fieldWysiwyg: {
+          type: 'object',
+          properties: { processed: { type: 'string' } },
+        },
```

Comment:
Due to how often this pattern comes up, we may want a `{ $ref: 'ProcessedString' }` or something similar.

Reply:
Yeah, I was thinking something similar too.
Just wrote a ticket for it: https://github.com/department-of-veterans-affairs/va.gov-team/issues/4302